tft.py

```diff
@@ -843,18 +783,22 @@ def main():
         )
         != "Start"
     ):
-        logger.warning("Intialization completed but aborting by user choice!")
+        logger.warning("Initialization completed but aborting by user choice")
         sys.exit(1)
 
     setup_hotkeys()
 
-    logger.info("Bot started, queuing up!")
+    LCU_INTEGRATION.connect_to_lcu()

```

Comment:
Since we have this running on startup, I wonder if it makes sense to introduce it automatically after the client is restarted, as well -- thoughts?

I see it's accounted for in the `catch` blocks of the other requests, but wondering if we should be pre-emptive after client restarts.

Reply:
Great suggestion, thanks!